asaps/models.py

```diff
@@ -109,31 +116,16 @@ def filter_note_type(client, csv_data, rec_obj, note_type, operation,
     return rec_obj
 
 
-def replace_str(rec_obj, fieldval, old_string, new_string):
+def replace_str(field_value, old_string, new_string):
```

Comment:
Is it necessary to have a function for this, or can you just use the Python string replace method `old_value.replace(old_string, new_string)` wherever you're calling this? i.e. where you call this in `filter_note_type`, could you instead just use `subnote['content'] = subnote['content'].replace(old_string, new_string)`?

Reply:
I've been thinking about that and I think I will be doing that in my next set of changes, it doesn't really make sense as a function unless it's assisting with the audit trail and I think I'm going to do that elsewhere in the code. And I would ditch the corresponding test if I get rid of it